_includes/install-script.md

```diff
@@ -1,48 +1,50 @@
-The source code for the script is open source, and can be found in the
-[`docker-install` repository on GitHub](https://github.com/docker/docker-install){:target="_blank" rel="noopener" class="_"}.
+Docker provides a convenience script at
+[https://get.docker.com/](https://get.docker.com/) to install Docker into
+development environments non-interactively. The convenience script isn't
+recommended for production environments, but it's useful for creating a
+provisioning script tailored to your needs. Also refer to the
+[install using the repository](#install-using-the-repository) steps to learn
+about installation steps to install using the package repository. The source
+code for the script is open source, and can be found in the
+[`docker-install` repository on GitHub](https://github.com/docker/docker-install){:target="_blank"
+rel="noopener" class="_"}.
 
+<!-- prettier-ignore -->
 Always examine scripts downloaded from the internet before running them locally.
 Before installing, make yourself familiar with potential risks and limitations
 of the convenience script:
 {:.warning}
 
 - The script requires `root` or `sudo` privileges to run.
 - The script attempts to detect your Linux distribution and version and
-  configure your package management system for you, and does not allow you to
-  customize most installation parameters.
+  configure your package management system for you.
+- The script doesn't allow you to customize most installation parameters.
 - The script installs dependencies and recommendations without asking for
   confirmation. This may install a large number of packages, depending on the
   current configuration of your host machine.
-- By default, the script installs the latest stable release of Docker, containerd,
-  and runc. When using this script to provision a machine, this may result in
-  unexpected major version upgrades of Docker. Always test (major) upgrades in
+- By default, the script installs the latest stable release of Docker,
+  containerd, and runc. When using this script to provision a machine, this may
+  result in unexpected major version upgrades of Docker. Always test upgrades in
   a test environment before deploying to your production systems.
-- The script is not designed to upgrade an existing Docker installation. When
+- The script isn't designed to upgrade an existing Docker installation. When
   using the script to update an existing installation, dependencies may not be
-  updated to the expected version, causing outdated versions to be used.
+  updated to the expected version, resulting in outdated versions.
 
 > Tip: preview script steps before running
 >
 > You can run the script with the `DRY_RUN=1` option to learn what steps the
-> script will execute during installation:
+> script will run when invoked:
 >
 > ```console
 > $ curl -fsSL https://get.docker.com -o get-docker.sh
 > $ DRY_RUN=1 sh ./get-docker.sh
```

Comment:
Should the `DRY_RUN` example include `sudo`?  It is required and shown in non-dry-run call below.